package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@oat-sa/tao-core-ui",
-    "version": "0.20.0",
+    "version": "0.20.1",
```

Comment:
Will need another bump when you're done, I think to `0.21.0` since this can be considered a feature.

Reply:
yes